model_analyzer/perf_analyzer/perf_analyzer.py

```diff
@@ -302,8 +300,9 @@ def _parse_perf_client_send_recv(self, section):
         if client_send_recv:
             client_send_recv = float(client_send_recv.group(1)) / 1e3
             return PerfClientSendRecv(value=client_send_recv)
-        raise TritonModelAnalyzerException(
-            'perf_analyzer output did not client send/recv time.')
+        logging.error(
```

Comment:
Maybe use warning instead of error?

Reply:
Changed to warning. @aramesh7 what do you think?

Reply:
Currently `-q` option on model analyzer silences everything below error, but I was wondering whether we should change that to allow warnings?

Reply:
Since this is not a critical error because the model analyzer can continue to function afterward, maybe it is okay not printing the warning if the user elects to silent it?

Reply:
Its fine as a warning for now.